test/controllers/api/v0/pages_controller_test.rb

```diff
@@ -11,12 +11,30 @@ class Api::V0::PagesControllerTest < ActionDispatch::IntegrationTest
   end
 
   # Regression
-  test 'should not include the `page` parameter multiple times in one paging link' do
-    # This error occurred when the requested URL already had a `page` param
-    get('/api/v0/pages?page=1')
+  test 'should not include the `chunk` parameter multiple times in one paging link' do
+    # This error occurred when the requested URL already had a `chunk` param
+    get('/api/v0/pages?chunk=1')
```

Comment:
This is fine, but two additional useful things you can do here:

- You can generate the URL or path with `api_v0_pages_path` or `api_v0_pages_url`.
- You can incorporate the query params in two ways:
    - An option on the `get` method:

        ```rb
        get(api_v0_pages_path, params: {chunk: 1})
        # or if you know it won’t collide with options that affect how the GET is done:
        get(api_v0_pages_path, chunk: 1)
        ```

    - As an option on the `api_v0_pages_path` or `_url` helper:

        ```rb
        get(api_v0_pages_path(params: {chunk: 1})
        # or if you know it won’t collide with more meaningful options:
        get(api_v0_pages_path(chunk: 1))
        ```

Neither of those are really a big deal, but I think they make the tests a little nicer because you can divorce the method being tested from its actual URL and you can avoid any URL encoding (not that any needs to be done for this particular argument and value).

Reply:
:+1: